code/datums/shuttles.dm

```diff
@@ -737,22 +755,27 @@
 /datum/map_template/shuttle/ruin/cyborg_mothership
 	suffix = "cyborg_mothership"
 	name = "Cyborg Mothership"
+	description = "A highly industrialised vessel designed for silicon operation infested with hivebots and space vines."
 
 /datum/map_template/shuttle/ruin/caravan_victim
 	suffix = "caravan_victim"
 	name = "Small Freighter"
+	description = "Small freight vessel, starts near blacked-out with 3 Syndicate Commandos and 1 Syndicate Stormtrooper, alongside a large hull breach."
 
 /datum/map_template/shuttle/ruin/pirate_cutter
 	suffix = "pirate_cutter"
 	name = "Pirate Cutter"
+	description = "Small pirate vessel with ballistic turrets. Spawns with 3 pirate simplemobs, one of which drops an energy cutlass."
 
 /datum/map_template/shuttle/ruin/syndicate_dropship
 	suffix = "syndicate_dropship"
 	name = "Syndicate Dropship"
+	description = "Light Syndicate vessel with laser turrets. Spawns with a Syndicate basic mob in the bridge."
 
 /datum/map_template/shuttle/ruin/syndicate_fighter_shiv
 	suffix = "syndicate_fighter_shiv"
 	name = "Syndicate Fighter"
+	description = "A small Syndicate vessel with exactly one tile of useful interior space and 4 laser turrets. Starts with a Syndicate basic mob in the pilot's seat, and extremely cramped."
```

Comment:
I would avoid calling them simplemobs/basic mobs here; it doesn't matter to an admin spawning them which one it is, and I don't expect whoever makes the pirates into basic mobs will remember to update this.

Reply:
Shrinked to just mobs.